storage/postgres/postgres.go

```diff
@@ -378,6 +385,7 @@ WHERE
   (? = '' OR operation_name = ?) AND
   DURATION(time) >= ? AND
   DURATION(time) <= ? AND
+  EXISTS ( SELECT 1 FROM spans AS sub_spans WHERE sub_spans.trace_id = spans.trace_id AND sub_spans.service_name IN (` + strings.Join(serviceConds, ", ") + `)) AND
```

Comment:
The zipkin ui does not appear to allow searching with no service name, so I feel this is a moot point for now.